src/main/java/com/example/lesson6/zad3/EvenOrOddNumbers.java

```diff
@@ -0,0 +1,19 @@
+package com.example.lesson6.zad3;
+
+public class EvenOrOddNumbers {
+
+
+    public static boolean getEvenNumbers(int number){
+        if (number % 2 == 0){
+            System.out.println("Liczba parzysta - " + number);
+        }
+        return number % 2 == 0;
+    }
+
+    public static boolean getOddNumbers (int number){
+        if (number % 2 > 0){
+            System.out.println("Liczba nieparzysta - " + number);
+        }
+        return number % 2 > 0;
```

Comment:
tutaj to zadziała, ale może lepiej utrzymać warunek podobnie jak wyżej czyli
` number % 2 != 0 `